thirdeye-ui/src/app/components/subscription-group-wizard/subscription-group-details/recipient-details/groups-editor/slack/slack.component.tsx

```diff
@@ -146,6 +159,40 @@ export const Slack: FunctionComponent<SlackProps> = ({
                         label={t("label.slack-url")}
                     />
                 </Grid>
+                <Grid container>
+                    <Controller
+                        control={control}
+                        name="notifyResolvedAnomalies"
+                        render={({ field: { name, value, onChange } }) => (
+                            <InputSection
+                                inputComponent={
+                                    <FormControlLabel
+                                        // eslint-disable-next-line max-len
+                                        // `control` needs to be passed on differently for checkbox
+                                        // eslint-disable-next-line max-len
+                                        // https://stackoverflow.com/questions/62291962/react-hook-forms-material-ui-checkboxes
```

Comment:
imo this is obvious and documented in the mui docs as well
can be removed.

Reply:
Ah, this comment was for my previous code at this (which did not work for the initial state)

Removing.